files/fr/web/html/link_types/preload/index.html

```diff
@@ -0,0 +1,222 @@
+  <li><code>script</code> : un fichier JavaScript ;</li>
+  <li><code>style</code> : une feuille de styles CSS ;</li>
+  <li><code>track</code> : un fichier WebVTT ;</li>
+  <li><code>worker</code> : un <i lang="en">web worker</i> JavaScript ou un <i lang="en">worker</i> partagé ;</li>
+  <li><code>video</code> : un fichier vidéo, typiquement intégré avec l'élément {{htmlelement("video")}}.</li>
+</ul>
+
+<div class="note">
+  <p><strong>Note :</strong> le préchargement de l'élément <code>video</code> est inclut dans la spécification <i lang="en">Preload</i> mais n'est pas encore implémenté par les navigateurs.</p>
+</div>
+
+<div class="note">
+<p><strong>Note :</strong> pour davantage de détails sur ces valeurs et sur la façon dont la spécification <i lang="en">Preload</i> prévoit de les traiter, consultez <a href="https://w3c.github.io/preload/#link-element-extensions"><i lang="en">link element extensions</i> (en anglais)</a>. Notez également que la liste complète des valeurs acceptées par l'attribut <code>as</code> est déterminé par la spécification <i lang="en">Fetch</i> — voir la spécification <a href="https://fetch.spec.whatwg.org/#concept-request-destination"><i lang="en">request destinations</i> (en anglais)</a>.</p>
+</div>
+
+<h2 id="including_a_mime_type">Inclure un type MIME</h2>
+
+<p>Les éléments <code>&lt;link&gt;</code> peuvent accepter un attribut {{htmlattrxref("type", "link")}}, contenant le type MIME de la ressource vers laquelle pointe le document. Ceci est tout spécialement utile lorsque l'on effectue un préchargement des ressources – le navigateur utilisera alors l'attribut <code>type</code> pour vérifier s'il prend en charge la ressource et la télécharger si c'est le cas, ou l'ignorer dans le cas contraire.</p>
+
+<p>Vous pouvez voir un exemple de ce fonctionnement dans notre vidéo d'exemple (voir le <a href="https://github.com/mdn/html-examples/tree/master/link-rel-preload/video">code source complet</a> ou la <a href="https://mdn.github.io/html-examples/link-rel-preload/video/">version exécutable en direct</a>), utilisant le bout de code proposé ci-dessous. À noter que si ce code ne lancera pas de préchargement effectif sur aucun navigateur – le préchargement des vidéos n'est encore implémenté sur aucun navigateur – cela permet d'illustrer le fonctionnement général du préchargement.</p>
+
+<pre class="brush: html">&lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+  &lt;title&gt;Exemple de préchargement de vidéo&lt;/title&gt;
+
+  &lt;link rel="preload" href="sintel-short.mp4" as="video" type="video/mp4"&gt;
+&lt;/head&gt;
+&lt;body&gt;
+  &lt;video controls&gt;
+    &lt;source src="sintel-short.mp4" type="video/mp4"&gt;
+    &lt;source src="sintel-short.webm" type="video/webm"&gt;
+    &lt;p&gt;Votre navigateur ne prend pas en charge les vidéos HTML5. Voici un &lt;a href="sintel-short.mp4"&gt;lien vers le fichier vidéo en remplacement&lt;/a&gt;.&lt;/p&gt;
```

Comment:
This could be splitted/newlines could be added to avoid horizontal scrolling in the box.